### PR TITLE
Cleanup imc tokenomics

### PR DESCRIPTION
**Pull request by Valdorff**
For astoneta's address verification see:

https://discord.com/channels/405159462932971535/1250191123146608721/1290129106997870657
image

### CODE REVIEW

File: RPIPs/RPIP-36.md [hunk 1]
```diff
@@ -60,26 +60,27 @@ Currently, there is no overview of the active committees within Rocket Pool, or
...
+| Date Changed | Members                                                                                                                    | Comment                                                                                                                                                                                          |
+|--------------|----------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   2024-06-10 | Jasper, Valdorff, peteris, waq, NonFungibleYokem, haloooloolo, samus, DrWorm, astoneta                                     | astoneta promoted as [2024 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x129eaa1779916b96fa1a34c7f9e24f87abad820c8fbe8ea2663f170891295e2e) runner-up after phils.eth stepped down |
```
Comment (orangesamus) [inline RPIPs/RPIP-36.md:79]:
```suggestion
|   2024-10-02 | Jasper, Valdorff, peteris, waq, NonFungibleYokem, haloooloolo, samus, DrWorm, astoneta                                     | astoneta promoted as [2024 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x129eaa1779916b96fa1a34c7f9e24f87abad820c8fbe8ea2663f170891295e2e) runner-up after phils.eth stepped down |
```

**Comment by orangesamus** [comment]
I thought we were going to leave all the tokenomics rework rpips as "living" instead of "final" so they can still be fixed if necessary as core team actually builds out implementation? I have some open issues already for things that still need updating: https://github.com/rocket-pool/RPIPs/issues/287

**Comment by Valdorff** [comment]
> I thought we were going to leave all the tokenomics rework rpips as "living" instead of "final" so they can still be fixed if necessary as core team actually builds out implementation (and set to "final" after our final vote on locked down specs prior to release)? I have some open issues already for things that still need updating: #287
> 
> https://discord.com/channels/405159462932971535/1215788197842255972/1256669893785554975 https://discord.com/channels/405159462932971535/1215788197842255972/1257234960918646914 https://discord.com/channels/405159462932971535/1215788197842255972/1258644562771968081 https://discord.com/channels/405159462932971535/1215788197842255972/1258656377971277904 https://discord.com/channels/405159462932971535/1215788197842255972/1259864846514655232

I'm up for that. If we do that, let's explicitly note that as the strategy in RPIP-49

**Comment by Valdorff** [comment]
Lol... RPIP-49 actually already explicitly calls out the Living thing:
> Run a [snapshot vote](https://vote.rocketpool.net/#/proposal/0xb0e1c82fdf83f31de7b8f84767a092fddfb21abb71d81ea3aeec9acdcf43902d) on the rework package as Living RPIPs

Thanks for the catch @orangesamus

**Review by orangesamus — APPROVED** [review]
:rocket:

**Pull request merged**